fuse.py

```diff
@@ -483,6 +491,11 @@ def _wrapper(func, *args, **kwargs):
             print_exc()
             return -EFAULT
 
+    def _handle_nopath(self, path):
```

Comment:
The `_decode_path` you suggested is trivial and provides no actual use. The method you are writing decodes the path and if it's None it does not blow up -- which **is** decoding the path and nothing else. If you have an issue with the name I gave you the opportunity to suggest something else: i.e. `_decode_optional_path(...)`. The naming you gave does not imply it is decoded and is not very descriptive other than path is likely None.